### PR TITLE
Fixes #191: List the RPM API for a provided module NSVC

### LINKED ISSUE #191: Create Python tutorial: List the RPM API for a provided module NSVC
Currently, the README.md and [modulemd.h](https://github.com/fedora-modularity/libmodulemd/blob/master/modulemd/v2/include/modulemd-2.0/modulemd.h) files have some limited guides for how to work with libmodulemd, but some real tutorials on API 2.0 would be ideal.

Create tutorial documentation (either extending modulemd.h annotations or by adding a new Tutorial section to the documentation) describing the following process with examples:

* Import the libmodulemd 2.0 API into a Python script
* Read repodata YAML from a file into an index
* Query the `Modulemd.ModuleIndex` for the module with a given name
* Query the `Modulemd.Module` for the `Modulemd.ModuleStream` associated with the provided NSVC (name-stream-version-context identifier).
* Read the RPM API from the `Modulemd.ModuleStream`

### PR DESCRIPTION
**Pull request by OrionStar25**
Fixes #191 

The asked piece of code is already in the current README. However I added 2 more definitions that you asked for over [here](https://github.com/fedora-modularity/libmodulemd/pull/202#pullrequestreview-209152985) 

Do tell me what else needs to be added.

### CODE REVIEW

**Comment by sgallagher** [comment]
@OrionStar25 Please see the description of that ticket. I want a full tutorial written. What's currently in there is a set of example code snippets.

Please write a descriptive tutorial around those.

**Comment by OrionStar25** [comment]
@sgallagher Please review.

File: README.md [hunk 1]
```diff
@@ -109,17 +111,37 @@ specification for a full list of information that can be retrieved.
...
 ## Python
 ```python
  module = merged_index.get_module ('modulename')
+```
+
+Then, query the Modulemd.Module for the Modulemd.ModuleStream associated with the provided NSVC
+(name-stream-version-context identifier).
+## C
+```C
+ModulemdModuleStream * stream = modulemd_module_get_stream_by_NSVC (module, "modulestream", 1, "deadbeef");
```
Comment (sgallagher) [inline README.md]: This needs updating, actually. We recently [replaced](https://github.com/fedora-modularity/libmodulemd/commit/c9708f5f49c28745b3ec7e204d493e66f0ad31ce) this function with `modulemd_module_get_stream_by_NSVCA()`. So we should update these docs not to reference it.

**Comment by OrionStar25** [comment]
@sgallagher  `modulemd_module_get_stream_by_NSVC()` is also being used to retrieve the modular runtime dependencies for a particular module NSVC. So I updated it there as well.

**Pull request merged**